Sources/NiceComponents/Theme/Config.swift

```diff
@@ -114,6 +116,15 @@ public struct Config {
             color: self.colorTheme.onSurface,
             theme: self.typeTheme.headline2
         )
+
+        // Set Shadow style
+
+        shadowStyle = ShadowStyle(
+            color: self.colorTheme.shadow,
```

Comment:
More of a subjective thing, but I'm curious where you picked these numbers from

Reply:
FWIW the Figma default seems to be x:0, y:4, blur: 4px, opacity: 0.15

Reply:
Oh I just randomly put them in. Figma default values seems like a good idea